modules/gltf/src/gltf/gltf-builder.js

```diff
@@ -4,6 +4,7 @@ import packBinaryJson from '../packed-json/pack-binary-json';
 import {assert} from '@loaders.gl/core';
 
 // Ideally we should just use KHR_draco_mesh_compression, but it requires saving uncompressed data?
+// TODO: until the ideal, we need to export these
```

Comment:
Nit: We should probably document them instead of exporting them. Maybe add a task for that?

Reply:
https://github.com/uber-web/loaders.gl/issues/65